app/(app)/get-started/_client.tsx

```diff
@@ -5,13 +5,19 @@ import Image from "next/image";
 import Link from "next/link";
 import { signIn } from "next-auth/react";
 import { useSearchParams } from "next/navigation";
-
-import { LockClosedIcon } from "@heroicons/react/20/solid";
+import { useEffect, useState } from "react";
 
 const GetStarted: NextPage = () => {
   const searchParams = useSearchParams();
   const callbackUrl = searchParams?.get("callbackUrl");
 
+  const [userEmail, setUserEmail] = useState<string>("");
+  const [isAlpha, setIsAlpha] = useState<boolean>();
+
+  useEffect(() => {
+    // toLowerCase can maybe be removed. Just want to make sure its not TRUE in Prod
+    setIsAlpha(process.env.ALPHA?.toLowerCase() === "true");
```

Comment:
Not sure if this is the correct way to go about alpha stuff. Further guidance needed